settings.js

```diff
@@ -13,6 +13,13 @@ var debugEnabled = process.env.DEBUG_ENABLED === 'false' ? false : true;
 var canaryEnabled = process.env.CANARY_ENABLED === 'true';
```

Comment:
Would like to rename this to avoid confusion, but no idea if people are relying on it. Will have to address it after the kayenta stuff is out for a while.